### ISSUE TITLE
Outdated

### ISSUE THREAD

**Issue by ijnrghjkdsmigywneig203** [QIN2DIM/undetected-playwright#2]
bot.sannysoft.com is quite outdated in terms of fingerprinting and detecting day to day bots. Using a site like: https://abrahamjuliot.github.io/creepjs/ or http://f.vision/ is a lot better and is up to date at detecting bots.

**Comment by QIN2DIM** [comment]
yeah, thanks for the issue. I plan to put more effort into learning more about this field.
